README.rst

```diff
@@ -1,30 +1,34 @@
-===========
+============
  Flask-IIIF
-===========
+============
 
 .. image:: https://travis-ci.org/inveniosoftware/flask-iiif.png?branch=master
     :target: https://travis-ci.org/inveniosoftware/flask-iiif
 .. image:: https://coveralls.io/repos/inveniosoftware/flask-iiif/badge.png?branch=master
     :target: https://coveralls.io/r/inveniosoftware/flask-iiif
 .. image:: https://pypip.in/v/Flask-IIIF/badge.png
-   :target: https://pypi.python.org/pypi/Flask-IIIF/
+    :target: https://pypi.python.org/pypi/Flask-IIIF/
 .. image:: https://pypip.in/d/Flask-IIIF/badge.png
-   :target: https://pypi.python.org/pypi/Flask-IIIF/
+    :target: https://pypi.python.org/pypi/Flask-IIIF/
 
 About
 =====
-Flask-IIIF is a Flask extension permitting to set up Shibboleth
-Single-Sign-On authentication in Flask based web applications.
+
+Flask-IIIF is a Flask extension permitting easy integration with the image
+IIIF API standards.
```

Comment:
What about: 

- Flask-IIIF is a Flask extension permitting easy integration with the International Image Interoperability Framework (IIIF) API standards.

(Here and also in `setup.py` in the `description` that currently says "Flask-IIIF multimedia extension")